src/data/markdown/docs/03 cloud/04 Integrations/04 Token.md

```diff
@@ -3,22 +3,45 @@ title: 'Token'
 excerpt: 'How to authenticate with k6 Cloud token'
 ---
 
-You'll need to authenticate to use k6 Cloud, whether it's for streaming results or running tests in the cloud. Your Auth Token enables the interaction with k6 Cloud using the k6 CLI or through the REST API. [Get your Auth Token](https://app.k6.io/account/api-token).
+You'll need to authenticate to use k6 Cloud, whether it's for streaming results or running tests in the cloud. Your API token enables the interaction with k6 Cloud using the k6 CLI or through the REST API.
 
-Below are some examples on how to utilize the token to authenticate.
+In k6 Cloud there are two ways how you can generate an API Token, via account or organization settings. These tokens differ in the kind of access they provide - [account level API token](#account-api-token) will grant access through your account with k6, while [organization level API token](#organization-api-token) will grant organization-wide access for actions not tied to a user.    
```

Comment:
```suggestion
You can generate in API token for k6 Cloud in two ways, each with different levels of access:
- [Account-level API tokens](#account-api-token) grant access through your account with k6
- [Organization-level API tokens](#organization-api-token) grant organization-wide access for actions not tied to a user.    
```

Reply:
> You can generate in API token for k6 Cloud in two ways, each with different levels of access:

I might be wrong as a non-native speaker, but a `level of access` sounds to me like with one token you get more permissions than with the other which is not necessarily true. It's just that there are different usages for the two tokens and the authentication happens on a different level, either through a user where the user's permissions are checked or globally for the organisation (than an admin must have created and provided the token).

Reply:
You're completely right! I changed it again using the terms you just used:

> You can generate two types of API tokens for k6 Cloud, depending on how you want to authenticate: